framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -264,6 +264,10 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 			blockHeaders,
 			validatorsHashPreimage,
 		);
+
+		if (!newCertificate && this._lastCertificate.height === 0) {
```

Comment:
I'm not sure if there is any test case but would be nice to have one if not present